src/Spinner.php

```diff
@@ -148,4 +145,16 @@ protected function eraseRenderedLines(): void
         $this->moveCursor(-999, -count($lines) + 1);
         $this->eraseDown();
     }
+
+    /**
+     * Clean up after the spinner.
+     */
+    public function __destruct()
+    {
+        parent::__destruct();
+
+        if (! empty($this->pid)) {
```

Comment:
Seems to me, that you want to do `if ($this->pid > 0)`?

Reply:
The destructor can be called before the fork occurs, so this property would not be initialised yet.

`isset($this->pid) && $this->pid > 0` would work, but at that point, it's effectively the same as `! empty($this->pid)` given that `$this->pid` is an int.